pom.xml

```diff
@@ -12,7 +12,7 @@
   <artifactId>rendezvous-service</artifactId>
   <name>SDO Rendezvous Service WebApp</name>
   <packaging>war</packaging>
-  <version>1.9</version>
+  <version>1.10</version>
```

Comment:
It would be a good idea to use SNAPSHOT in version number during development cycle and move on to use the definitive version number at the time of release.

1.10-SNAPSHOT

Reply:
Done